src/remoted/request.c

```diff
@@ -169,6 +177,12 @@ void * req_dispatch(req_node_t * node) {
             rem_inc_send_request(agentid);
         }
 
+        if (isForceReconnect) {
+            if (OS_SendSecureTCP(node->sock, strlen("ack"), "ack") != 0) {
```

Comment:
What if you add a `const char *` for the `ack` string here https://github.com/wazuh/wazuh/pull/15281/files#diff-d03df09093a8316731cbee56730a13bed4c9b7c8f1d14d28481931f8beb1b671R31?